plugin/openassetio_manager_bal/BasicAssetLibraryInterface.py

```diff
@@ -179,6 +191,12 @@ def resolve(
                     trait_data = entity.traits.get(trait)
                     if trait_data is not None:
                         self.__add_trait_to_traits_data(trait, trait_data, result)
+
+                if VersionTrait.kId in traitSet:
+                    version_trait = VersionTrait(result)
+                    version_trait.setStableTag(str(entity.version))
+                    version_trait.setSpecifiedTag(str(entity_info.version or VERSION_TAG_LATEST))
```

Comment:
[question] If I understand correctly from my quick scan, a BAL entity reference signals it is the latest version by simply having no `v` parameter at all. 

In this case putting `"latest"` into the `specifiedTag` is a misnomer, and should be blank instead since no tag is "specified" in the entity reference.

It's certainly more useful to give back "latest" rather than an empty string. I guess we just have to ensure it's documented properly in MediaCreation - currently a little ambiguous:

```

              If the reference itself does not contain a version
              specifier and relies on dynamic behaviour, then this should
              be left empty.
```

Reply:
Looking good in that other PR, resolving this.